apps/test/unit/lib/kits/maker/commandsTest.js

```diff
@@ -4,19 +4,20 @@ import sinon from 'sinon';
 import {
   analogRead,
   analogWrite,
+  boardConnected,
   digitalRead,
   digitalWrite,
   injectBoardController,
   onBoardEvent,
   pinMode
 } from '@cdo/apps/lib/kits/maker/commands';
-import CircuitPlaygroundBoard from '@cdo/apps/lib/kits/maker/CircuitPlaygroundBoard';
+import FakeBoard from '@cdo/apps/lib/kits/maker/FakeBoard';
```

Comment:
Hooray for interfaces in JavaScript! Where is the `MakerBoard` interface defined?